docker/fc34/kvm.Dockerfile

```diff
@@ -61,6 +61,14 @@ RUN \
     unzip \
     valgrind \
     wget \
+    autoconf \
+    automake \
+    libtool \
+    g++ \
+    vim \
+    iperf \
+    crash \
+    zstd \
```

Comment:
this is the development convenience.

Maybe it'd be cool to allow user the ability to provide his own docker file that will incrementally append needed things to the already existing image,

Reply:
It was always in back of my mind, but didn't investigate how to do it without rebuilding all images.

Reply:
Can you just apply something on top of the existing image?
Like running one additional docker file over the existing image extending it and making the new one the “current “?